lib/commentOnOpenedPullRequest.js

```diff
@@ -33,13 +33,17 @@ function commentOnOpenedPullRequest(body, repositorySettings) {
 commentOnOpenedPullRequest._implementation = function (pullRequestFilesUrl, pullRequestCommentsUrl, repositorySettings, userName, repositoryUrl) {
     var claEnabled = defined(repositorySettings.claUrl);
     var askForCla = false;
+    var errorCla;
     return repositorySettings.fetchSettings()
         .then(function () {
             return commentOnOpenedPullRequest._askForCla(userName, repositorySettings);
         })
         .then(function (result) {
             askForCla = result;
         })
+        .catch(function(error) {
+            errorCla = error.toString();
```

Comment:
It looks like we treat this as a boolean, so shouldn't this just be `errorCla = true`?

Reply:
> You can use the if helper to conditionally render a block. If its argument returns false, undefined, null, "", 0, or [], Handlebars will not render the block.

I do print the error, so I'm using the string if it's defined.

Reply:
Fair enough.